Rakefile

```diff
@@ -87,3 +87,54 @@ namespace :students do
     end
   end
 end
+
+namespace :comments do
+  task :migrate do
+    Mongo::Logger.logger.level = ::Logger::FATAL
+    module Classroom::Collection::Comments
+      extend Mumukit::Service::Collection
+      def self.mongo_collection_name
+        :comments
+      end
+      def self.mongo_database
+        Classroom::Database
+      end
+    end
+    class Classroom::Collection::ExerciseStudentProgress < Classroom::Collection::CourseCollection
```

Comment:
@aguspina @fedescarpa have you already migrated those comments? Can we remove this code now?